### DISCUSSION TITLE
[@types/node] promiseHooks types not exposed

### DISCUSSION THREAD

**Discussion by kwasimensah** [DefinitelyTyped/DefinitelyTyped#58897] [Issues with a @types package] [answered]
https://nodejs.org/docs/latest-v16.x/api/v8.html#promise-hooks isn't exposed to Typescript

[a comment by an automated account is not shown]

**Comment by MayCXC** [comment, marked as answer]
hopefully https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65274 can help :)

**Reply by peterblazejewicz** [reply]
merged: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65274#issuecomment-1522443318
